src/components/Notice/Header.tsx

```diff
@@ -0,0 +1,43 @@
+import { useNavigate } from 'react-router-dom';
+
+import styled from 'styled-components';
+
+type HeaderProps = {
+    children: React.ReactNode;
+    shadow: 'true' | 'false';
+}
+
+const HeaderContainer = styled.div <{ shadow: string }>`
+display: flex;
+align-items: center;
+padding-inline: ${(props) => props.theme.sizes.contentPadding};
+padding-block: 1.4em;
+box-shadow: ${(props) => (props.shadow === 'true' ? '0px 0.3rem 0.3rem 0px rgba(0, 0, 0, 0.05)' : 'none')};
+
+span{
+  margin: 0 auto;
+  font-weight: 700;
+  letter-spacing: -0.04638rem;
+  flex-direction: column;
+  padding-right: 11px;
+}
+
+button{
+  background: none;
+  border: none;
+}
+`;
+
+export default function Header({ children, shadow }: HeaderProps) {
+  const navigate = useNavigate();
+  return (
```

Comment:
사소하지만 개행이 하나 들어가면 좋을 것 같네요~
```suggestion
  const navigate = useNavigate();
  
  return (
```